### DISCUSSION TITLE
我买了一年的专业版，为啥我蓝灯上还是显示的免费版，无法使用Pro版

### DISCUSSION THREAD

**Discussion by heidelbergkai** [getlantern/lantern-forum-cn#755] [Q&A]
我三天前买了一年的专业版，为啥我蓝灯上还是显示的免费版，无法使用Pro版。
我的账号是812862967@qq.com
请帮我一下。
![payment](https://github.com/user-attachments/assets/97e5ed33-2b35-4766-b841-ea438c08cd73)

**Comment by MaxMKMax** [comment]
@heidelbergkai 您好. 我已经帮您的问题提交上去了. 这两天请查看这个帖子或邮箱来获取激活码. 谢谢!

**Reply by heidelbergkai** [reply]
没有收到任何激活码？怎么回事？
